tbx/work/models.py

```diff
@@ -44,21 +35,16 @@ class WorkPageScreenshot(Orderable):
     ]
 
 
-class WorkPageAuthor(Orderable):
-    page = ParentalKey("work.WorkPage", related_name="authors")
-    author = models.ForeignKey(
-        "people.Author", on_delete=models.CASCADE, related_name="+"
-    )
-
-    panels = [
-        FieldPanel("author"),
-    ]
-
+class HistoricalWorkPage(SocialFields, Page):
+    """
+    This represents Work Pages as they were prior to the 2024
+    rebuild of the site. It is kept here for historical purposes.
```

Comment:
Suggestion: change the last sentence to "It is kept here to display the older work pages that were created prior to 2024."

Reply:
done in 98f6fe4